tests/__init__.py

```diff
@@ -0,0 +1,12 @@
+import logging
+from tests.conftest import params, Helpers
+
+
+logging.basicConfig(level=logging.ERROR)  # external logging level
+log: logging.Logger = logging.getLogger('onyo')  # internal logging level
+log.setLevel(level=logging.INFO)
+
+__all__ = [
+    params,
+    Helpers
```

Comment:
Add a `,` at the end here

Reply:
Lol, fair enough, though I want to add for the record, it does not have to literally be everywhere in the end, I was just again removing them in the end where they were already existing ;)

But I added it, so thanks for having a look!